PocketKit/Sources/PocketKit/Services.swift

```diff
@@ -11,7 +11,7 @@ import SharedPocketKit
 import Kingfisher
 
 struct Services {
-    static let shared = Services()
+    static let shared: Services = { Services() }()
```

Comment:
lazy loading

Reply:
…there's another version of lazy loading other than `lazy var`? I thought this is just using the result of a closure to declare a `let`.

Reply:
> Global constants and variables are always computed lazily, in a similar manner to [Lazy Stored Properties](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/properties#Lazy-Stored-Properties). Unlike lazy stored properties, global constants and variables don’t need to be marked with the lazy modifier.

> Local constants and variables are never computed lazily.

TIL. https://docs.swift.org/swift-book/documentation/the-swift-programming-language/properties